docs/version_patcher/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"compilerOptions": {
+		"target": "ESNext",
+		"module": "ESNext",
+		"moduleResolution": "Node",
+		"lib": ["DOM", "ESNext"],
+		"outDir": "lib",
+		"allowSyntheticDefaultImports": true
+	},
+	"include": ["src/**/*.ts"]
+}
```

Comment:
Do we really need this? Isn't this a TypeScript related file and we aren't using TypeScript?

Reply:
oh, good catch, I removed the other TS stuff